NuKeeper/Engine/Packages/ExistingBranchFilter.cs

```diff
@@ -24,7 +24,7 @@ public ExistingBranchFilter(ICollaborationPlatform collaborationPlatform, INuKee
             try
             {
                 var branchName = BranchNamer.MakeSinglePackageName(packageUpdateSet);
-                return await _collaborationPlatform.RepositoryBranchExists(pushFork.Owner, pushFork.Name, branchName);
+                return await _collaborationPlatform.RepositoryBranchExists(pushFork.Owner, pushFork.Name, branchName) == false;
```

Comment:
I'm not a fan of `== false` 
I would even put it on 2 lines if it makes is easier to deal with the await and the conditional, e.g.
```
var branchExists = await _collaborationPlatform.RepositoryBranchExists(pushFork.Owner, pushFork.Name, branchName);
return ! branchExists;
```